CoreTableView/src/au/gov/asd/tac/constellation/views/tableview/state/TableStateIOProvider.java

```diff
@@ -68,7 +68,6 @@ public void readObject(final int attributeId, final int elementId, final JsonNod
                     state.columns.add(cs);
                 } catch (IllegalArgumentException ex) {
                     LOGGER.log(Level.SEVERE, "Segment value {0} illegal", segmentText);
```

Comment:
can you pass ```ex``` through to the logger too?

Reply:
Change has now been made